src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.springboot.datajpa.model;
+
+import java.math.BigDecimal;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+
+@Entity
+@Table(name = "products")
+@Data
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private BigDecimal price;
+    @ManyToOne

```

Comment:
one product can have one category
```suggestion
    @OneToOne

```